bot.py

```
@@ -0,0 +1,248 @@
+    async def take_control(ctx, control_token: str) -> None:
+        """Take control of the bot
+        
+        Args:
+            control_token (str): The control token
+        """
+        user_id = str(ctx.author.id)
+        if bot_control.request_control(user_id, control_token):
+            # Control granted silently
+            return
+        else:
+            await ctx.send("Invalid control token or you are not authorized")
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)